content/getting-started/usage.md

```diff
@@ -233,11 +234,11 @@ By using Hugo's server in production, you are able to deploy just the source fil
 
 Interested? Here are some great tutorials contributed by Hugo users:
 
-* [hugo, syncthing](http://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
+* [hugo, syncthing](https://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
```

Comment:
In other words, can we possibly delete this entire section of the page? 

http://localhost:1313/getting-started/usage/#using-hugo-s-server-in-production